funnel/templates/membership_add_invite_email.md

```diff
@@ -0,0 +1,3 @@
+**{{ invited_by.pickername }}** invited you to join **{{ project.title }}**.
```

Comment:
`granted_by` for consistency, although it is fairly contorted to say you've been granted an invite.